### ISSUE TITLE
PaginatedResponse not working correctly

### ISSUE THREAD

**Issue by iamkubi** [iamkubi/pydactyl#47]
The prints in this code have the same result:

```
    response = api.nodes.list_nodes()
    for page in response:
        print ('page: {}'.format(page))
        for node in page:
            print('node: {}'.format(node))
```

Must be a bug in PaginatedResponse.

**Comment by iamkubi** [comment]
This is more or less intended.  When creating an iterator of a PaginatedResponse it re-initializes the data and the string method returns the same thing.  To iterate over the items in a paginated response it needs to loop over `page.data`.

**Comment by iamkubi** [comment]
14 days ago me was not very bright.  Jan 10th me knew what I was talking about.

**Comment by iamkubi** [comment]
Fixed by b81b276
